tests/TDD.md

```diff
@@ -0,0 +1,91 @@
+## Test Driven Development (TDD) for ODO
+
+What’s TDD?  In simple terms, WRITE TESTS FIRST
+
+#### TDD Workflow
+
+Under TDD workflow, we write tests before writing feature code
+The tests written will serve as a blueprint for writing feature code
+
+1. Add a test
+1. Run all tests and see if the new test fails
```

Comment:
although this is auto generated on GitHub, you need to change the numbers to 1,2,3,4,etc.